ext/standard/basic_functions.stub.php

```diff
@@ -923,7 +923,7 @@ function lchgrp(string $filename, string|int $group): bool {}
 function chmod(string $filename, int $mode): bool {}
 
 #if HAVE_UTIME
-function touch(string $filename, ?int $time = null, ?int $atime = null): bool {}
+function touch(string $filename, ?int $mtime = null, ?int $atime = null): bool {}
```

Comment:
I've kept these terms because we have filemtime() and fileatime() and also return these keys from stat/etc. Not really opposed to more explicit names though.

Reply:
ok, this reasoning seems sensible for me as well, so let's keep it as-is.